documentation/src/docs/asciidoc/user-guide/writing-tests.adoc

```diff
@@ -2927,21 +2927,30 @@ In addition to controlling the execution mode using the `{Execution}` annotation
 Jupiter provides another annotation-based declarative synchronization mechanism. The
 `{ResourceLock}` annotation allows you to declare that a test class or method uses a
 specific shared resource that requires synchronized access to ensure reliable test
-execution. The shared resource is identified by a unique name which is a `String`. The
-name can be user-defined or one of the predefined constants in `{Resources}`:
+execution. The shared resource is identified by a unique name which is a `String`.
+The name can be user-defined or one of the predefined constants in `{Resources}`:
 `SYSTEM_PROPERTIES`, `SYSTEM_OUT`, `SYSTEM_ERR`, `LOCALE`, or `TIME_ZONE`.
 
-If the tests in the following example were run in parallel _without_ the use of
-{ResourceLock}, they would be _flaky_. Sometimes they would pass, and at other times they
-would fail due to the inherent race condition of writing and then reading the same JVM
-System Property.
+Since Junit Jupiter 5.12 `{ResourceLock}` annotation has 'providers' attribute
```

Comment:
Since the User Guide is versioned, we usually avoid mentioning specific versions when features were introduced. Thus, please remove that bit.

Reply:
Got it. Done